packages/design-system/src/components/Dropdown/Primitive/DropdownShell.module.scss

```
@@ -0,0 +1,22 @@
+@use '~@talend/design-tokens/lib/tokens';
+
+.dropDownShell {
```

```suggestion
.dropdownShell {
```
to be consistent with `dropdownTitle`